<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>download_tool</class>
 <widget class="QMainWindow" name="download_tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="io_panel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>401</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Download">
     <attribute name="title">
      <string>Download</string>
     </attribute>
     <widget class="QListWidget" name="fileList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>361</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="download_browse_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>460</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="download_confirm_btn">
      <property name="geometry">
       <rect>
        <x>142</x>
        <y>460</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="download_cancel_btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>460</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="download_progress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="download_msg_box">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Upload">
     <attribute name="title">
      <string>Upload</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>311</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; &quot;&gt;Choose a file to upload&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="upload_progress">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="upload_url">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="upload_browse_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upload_confirm_btn">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="upload_cancel_btn">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="upload_msg_box">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>361</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
